unity-client/Assets/Scripts/MainScripts/DCL/Components/ParametrizedShapes/Tests/ParametrizedShapesTests.cs

```diff
@@ -6,363 +6,302 @@
+        var meshName = scene.entities[entityId].gameObject.GetComponentInChildren<MeshFilter>().mesh.name;
+        Assert.AreEqual("DCL Sphere Instance", meshName);
+        yield break;
+    }
 
-            var meshName = scene.entities[entityId].gameObject.GetComponentInChildren<MeshFilter>().mesh.name;
-            Assert.AreEqual("DCL Plane Instance", meshName);
-            yield break;
-        }
 
+    [UnityTest]
+    public IEnumerator CylinderShapeUpdate()
+    {
+        string entityId = "5";
+        TestHelpers.InstantiateEntityWithShape(scene, entityId, DCL.Models.CLASS_ID.CYLINDER_SHAPE, Vector3.zero);
 
-        [UnityTest]
-        public IEnumerator PlaneShapeUpdateWithUVs()
-        {
-            float[] uvs = new float[] {0, 0.75f, 0.25f, 0.75f, 0.25f, 1, 0, 1, 0, 0.75f, 0.25f, 0.75f, 0.25f, 1, 0, 1};
-
-            DecentralandEntity entity;
-
-            PlaneShape plane = TestHelpers.InstantiateEntityWithShape<PlaneShape, PlaneShape.Model>(
-                scene,
-                DCL.Models.CLASS_ID.PLANE_SHAPE,
-                Vector3.zero,
-                out entity,
-                new PlaneShape.Model()
-                {
-                    height = 1,
-                    width = 1,
-                    uvs = uvs
-                });
-
-            yield return plane.routine;
-
-            Assert.IsTrue(entity != null);
-            Assert.IsTrue(plane != null);
-            Assert.IsTrue(plane.currentMesh != null);
-            CollectionAssert.AreEqual(Utils.FloatArrayToV2List(uvs), plane.currentMesh.uv);
-        }
-
-        [UnityTest]
-        public IEnumerator CylinderShapeUpdate()
-        {
-            string entityId = "5";
-            TestHelpers.InstantiateEntityWithShape(scene, entityId, DCL.Models.CLASS_ID.CYLINDER_SHAPE, Vector3.zero);
+        var meshName = scene.entities[entityId].gameObject.GetComponentInChildren<MeshFilter>().mesh.name;
+        Assert.AreEqual("DCL Cylinder Instance", meshName);
+        yield break;
+    }
 
-            var meshName = scene.entities[entityId].gameObject.GetComponentInChildren<MeshFilter>().mesh.name;
-            Assert.AreEqual("DCL Cylinder Instance", meshName);
-            yield break;
-        }
+    [UnityTest]
+    public IEnumerator ConeShapeUpdate()
+    {
+        string entityId = "4";
+        TestHelpers.InstantiateEntityWithShape(scene, entityId, DCL.Models.CLASS_ID.CONE_SHAPE, Vector3.zero);
 
-        [UnityTest]
-        public IEnumerator ConeShapeUpdate()
-        {
-            string entityId = "4";
-            TestHelpers.InstantiateEntityWithShape(scene, entityId, DCL.Models.CLASS_ID.CONE_SHAPE, Vector3.zero);
+        var meshName = scene.entities[entityId].gameObject.GetComponentInChildren<MeshFilter>().mesh.name;
 
-            var meshName = scene.entities[entityId].gameObject.GetComponentInChildren<MeshFilter>().mesh.name;
+        Assert.AreEqual("DCL Cone50v0t1b2l2o Instance", meshName);
```

Comment:
The name "_Cone50v0t1b2l2o_"... where does it come from? Is it sure to check these kind of specific names in our tests?

Reply:
These tests are ancient, so most likely not.